modules/spo-recording-profiles.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-recording-profiles_{context}"]
+= Recording profiles from workloads
+
+The Security Profiles Operator can record system calls with `ProfileRecording` objects, making it easier to create baseline profiles for applications.
+
+When using the log enricher for recording {type} profiles, verify the log enricher feature is enabled. See _Additional resources_ for more information.
+
+[NOTE]
+====
+A container with `privileged: true` security context restraints prevents log-based recording. Privileged containers are not subject to {type} policies and log-based recording makes use of a special {type} profile to record events.
+====
+
+.Procedure
+
+. Create a `ProfileRecording` containing a `recorder: logs` variable:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileRecording
+metadata:
+  name: test-recording
+spec:
+  kind: {kind}
+  recorder: logs
+  podSelector:
+    matchLabels:
+      app: my-app
+----
+
+. Create a workload to record:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-pod
+  labels:
+    app: my-app
+spec:
+  containers:
+    - name: nginx
+      image: quay.io/security-profiles-operator/test-nginx:1.19.1
+    - name: redis
+      image: quay.io/security-profiles-operator/redis:6.2.1
+----
+
+. Confirm the pod is in a `Running` state by entering the following command:
++
+[source,terminal]
+----
+$ oc -n security-profiles-operator get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME     READY   STATUS    RESTARTS   AGE
+my-pod   2/2     Running   0          18s
+----
+
+. The enricher indicates that it receives audit logs for those containers:
++
+[source,terminal]
+----
+$ oc -n security-profiles-operator logs --since=1m --selector name=spod log-enricher
```

Comment:
Need to upate to 
$ oc -n openshift-security-profiles  logs --since=1m --selector name=spod -c log-enricher
```
$ oc -n openshift-security-profiles         logs --since=1m --selector name=spod log-enricher
error: only a selector (-l) or a POD name is allowed
See 'oc logs -h' for help and examples
$ oc -n openshift-security-profiles         logs --since=1m --selector name=spod -c log-enricher
I1208 14:25:55.862248  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=39 "syscallName"="getpid" "timestamp"="1670509555.860:9822782" "type"="seccomp"
I1208 14:25:55.862406  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=257 "syscallName"="openat" "timestamp"="1670509555.860:9822783" "type"="seccomp"
I1208 14:25:55.862553  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=0 "syscallName"="read" "timestamp"="1670509555.860:9822784" "type"="seccomp"
I1208 14:25:55.862668  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=3 "syscallName"="close" "timestamp"="1670509555.860:9822785" "type"="seccomp"
I1208 14:25:55.862835  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=232 "syscallName"="epoll_wait" "timestamp"="1670509555.860:9822786" "type"="seccomp"
I1208 14:25:55.962428  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=39 "syscallName"="getpid" "timestamp"="1670509555.960:9822787" "type"="seccomp"
I1208 14:25:55.962589  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=257 "syscallName"="openat" "timestamp"="1670509555.960:9822788" "type"="seccomp"
I1208 14:25:55.962762  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=0 "syscallName"="read" "timestamp"="1670509555.960:9822789" "type"="seccomp"
I1208 14:25:55.963020  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=3 "syscallName"="close" "timestamp"="1670509555.960:9822790" "type"="seccomp"
I1208 14:25:55.963155  211126 enricher.go:446] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="ip-10-0-212-96.us-east-2.compute.internal" "pid"=234142 "pod"="my-pod" "syscallID"=232 "syscallName"="epoll_wait" "timestamp"="1670509555.960:9822791" "type"="seccomp"

```